virtool/tasks/runner.py

```diff
@@ -12,45 +11,90 @@
 from sentry_sdk import capture_exception
 
 
+async def shutdown_task_runner(app: Application):
+    """
+    Raise an `asyncio.CancelledError` in the running task runner.
+
+    :param app: Current running application.
+    """
+    await app["task_runner"].close(timeout=float("inf"))
+
+
+SHUTDOWN_TIMEOUT: int = 600
```

Comment:
Needs docstring. Constants should be defined near the top of the module after imports.
```suggestion
SHUTDOWN_TIMEOUT: int = 600
"""Something here"""
```